internal/output/prometheusrw/remotewrite/remotewrite_test.go

```diff
@@ -67,15 +67,29 @@ func TestOutputConvertToPbSeries(t *testing.T) {
 			Time:  time.Date(2022, time.September, 1, 0, 0, 0, 0, time.UTC),
 			Value: 7,
 		},
+		metrics.Sample{
+			TimeSeries: metrics.TimeSeries{
+				Metric: registry.MustNewMetric("metric4", metrics.Trend),
+				Tags: registry.RootTagSet().WithTagsFromMap(tagset.Map()).
+					With("We_need", "value").
+					With("a_long", "value").
+					With("list_of", "value").
+					With("tags_to", "value").
+					With("reproduce_5175", "value"),
```

Comment:
While [this](https://github.com/grafana/k6/issues/5175#issuecomment-3287788875) comment was super illuminating, it required me some extra time to figure out while the test was still passing with the old code.

Turns out to be the case that, the issue was not only reproducible when there was a label that starts with a capital letter, but only when the total amount of labels is also large enough. That's why adding a new tag starting with a capital letter wasn't enough.

Note that in the real-world scenario reported in #5175, the user is only adding one tag (starting with capital letter!), but k6 adds some more (enough!) when reporting the HTTP metrics (like `proto`,  `url`, etc).